app/error.tsx

```diff
@@ -0,0 +1,34 @@
+'use client';
+
+import { Button, Center, Flex, Heading, Text } from '@chakra-ui/react';
+
+export default function Error({
+  error,
+  reset,
+}: {
+  error: Error;
+  reset: () => void;
+}) {
+  return (
+    <Center width="100%">
+      <Flex
+        flexDirection="column"
+        alignItems="center"
+        gap="24px"
+        maxWidth="1500px"
```

Comment:
```suggestion
        maxWidth={layoutConfig.maxWidth}
```
`import layoutConfig from '@/config/layout';`

Reply:
Will update this, thanks